css/style.css

```diff
@@ -274,14 +308,22 @@ a {
   padding-top: 2.5em;
 }
 
-.pad-bottom-30 {
-  padding-bottom: 30px;
+.pad-top-4em {
+  padding-top: 4em;
 }
 
 .pad-top-7em {
   padding-top: 7em;
 }
 
+.pad-bottom-30 {
+  padding-bottom: 30px;
+}
+
+.pad-bottom-3em {
+  padding-bottom: 3.5em;
+}
+
 .pad-bottom-2em {
   padding-bottom: 2.5em;
 }
```

Comment:
- [ ] DRY stands for "Don't Repeat Yourself." The main idea with DRY in CSS is to avoid repeating the same bits of code in our style sheet. You are repeating your code multiple times, Let us fix this issue in this way. 
    - Group reusable css properties together.
    - Name these groups logically.
    - Add your selectors to the various css groups.

Reply:
Like I said earlier I tried to group these to make them reusable and you still have complaints about them.